instance.tf

```diff
@@ -0,0 +1,64 @@
+resource "aws_instance" "web" {
+  ami           = "ami-0022f774911c1d690"
```

Comment:
This will only work in a single region. In case we want to deploy to another region the code will fail.
Please used aws_ami data source so the code is more universal.
Here is the documentation:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ami

Here are examples:
https://github.com/vladimirmukhin/data_source_example/blob/main/main.tf